deployment/kustomizations/kustomization_example.yaml

```diff
@@ -0,0 +1,60 @@
+##### Dependancies used in this kustomization:
```

Comment:
Since `kustomization_example.yaml` & `kustomization.yaml` share the same contents. I'm thinking we should just keep `kustomization.yaml`? If users want to edit `kustomization.yaml`, they can just copy it.

Reply:
They do share same content, except some things are simply commented. But feel free to remove the kustomization_example.yaml and keep only the kustomization.yaml